captum/attr/_core/integrated_gradients.py

```diff
@@ -27,15 +30,21 @@ def __init__(self, forward_func):
 
     def attribute(
         self,
-        inputs,
-        baselines=None,
-        target=None,
-        additional_forward_args=None,
-        n_steps=50,
-        method="gausslegendre",
-        internal_batch_size=None,
-        return_convergence_delta=False,
-    ):
+        inputs: Union[Tensor, Tuple[Tensor, ...]],
+        baselines: Optional[
+            Union[Tensor, int, float, Tuple[Union[Tensor, int, float], ...]]
```

Comment:
Should we define a type alias for this, e.g. 

```
Baseline = Union[Tensor, int, float]
```

?

We can create a new file just for defining Captum types.

Reply:
Yes, this is a good point! I debated this a bit when I wrote the type hints initially. There are some repetitive patterns like these, which would be more concise to move to a separate file, but the advantage of having it here is that someone just reading the signature has the explicit type information readily available without having to look at any other files to find the type definition of aliases. The type information is often provided in the docstrings, but sometimes not as explicitly as the type hints, for instance, in this case of baselines. I'm open to either approach, what do you think @edward-io @NarineK ?

Reply:
I agree that `Baseline` will improve readability but users need to navigate to `Baseline ` type to figure out what it really is. We could have a comment explaining what it is but that might create redundancy with the documentation. How do other well known python libraries that support type hints do ? Maybe we can do a short research on that ?